build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -235,6 +249,27 @@ service ActionCache {
 // `instance_name` is the instance name (see above), and `hash` and `size` are
 // the [Digest][build.bazel.remote.execution.v2.Digest] of the blob.
```

Comment:
The same rules about verifying data and retrying apply here, too.

Reply:
I think I covered this in the restructuring.